### DISCUSSION TITLE
Where are information of key and url in config

### DISCUSSION THREAD

**Discussion by t-anh** [ONLYOFFICE/DocumentServer#1643] [Q&A]
I have already installed document server docker and run the example test successfully.
http://documentserver/example/
And I can edit files on the test example page
http://documentserver/example/editor?type=desktop&fileName=new.docx
At [Basic concept](https://api.onlyoffice.com/editors/basic), the config object of the target HTML file has a key and URL.
But I don't know how to find that information.
Where I can find information on key and URL to initialize config object?
> config = {
>     "document": {
>         "fileType": "docx",
>         "key": "Khirz6zTPdfd7",
>         "title": "Example Document Title.docx",
>         "url": "https://example.com/url-to-example-document.docx"
>     },
>     "documentType": "word",
>     "editorConfig": {
>         "callbackUrl": "https://example.com/url-to-callback.ashx"
>     }
> };

**Comment by ShockwaveNN** [comment]
This help entry?

https://api.onlyoffice.com/editors/config/document#key

In short - `key` is sequence of random symbols
URL is the path to file stored on DMS (document management system)

What's your end goal? Integrate into some own DMS? Or just use it with popular ones (like Nextcloud) - in that case you don't need to know all the deails

**Comment by t-anh** [comment]
Thank you for your quick reply!
I want to use OnlyOffice to edit files on both desktop and mobile.
I intend to implement the following step:

1. Download file into /var/lib/onlyoffice/documentserver-example/files/{browser-ip-address}
2. Edit file by open below URL at browser
http://{document-server}/example/editor?type=desktop&mode=edit&fileName=new.docx
3. After editing, upload file into destination.

I want to hide the Download and the Print menu option from the File menu, and change the OnlyOffice logo.
So I want to change the config to hide them.
What should I do?

I also want to create sub folder at /var/lib/onlyoffice/documentserver-example/files/{browser-ip-address} and edit file of subfolder.
Which URL should be used to edit the file in the subfolder?

**Comment by ShockwaveNN** [comment]
I think for you case better use some DMS, like Nextcloud
Please note that integrated `/example` is not for production use, but for development and checking your own integration, which as far as I understand you have none

>  and change the OnlyOffice logo.

You can change our logo only if you have special license with customization, it's not possible in free edition

And if you got license - I think you better contact support@onlyoffice.com and they answer all you question since seems this is not an issue, but rather a question - I moving this to dicsussions

**Reply by t-anh** [reply]
@ShockwaveNN 
Thank you for your reply!
I will contact support@onlyoffice.com.
